engine/events/overworld.asm

```diff
@@ -323,7 +323,7 @@ Script_UseFlash:
 	end
 
 UnknownText_0xc8f3:
-	text_far UnknownText_0x1c0609
+	text_far Text_UseFlash
```

Comment:
Also make sure the text_far labels use the same name as their parent text script counterpart.

Reply:
Thanks for pointing me to the clean up, couldn't quite remember which document pointed that out, I'll get to work on cleaning it up properly. And my main focus on this part was going through the common text, but getting that out of the way is definitely in the plans.

Reply:
Considering these are simply pointers to the common text it's probably easier to do it right now than have to clean it up later, but if you wish to leave this for a future PR, I guess that's fine.

Reply:
To be honest, most things outside of the common text asm I wasn't aware of, because I was using `sed` and `grep` heavily.